### PR TITLE
Add new detailed errors to executor.proto

### PR DESCRIPTION
**Pull request by fractasy**

### CODE REVIEW

**Review by ToniRamirezM — APPROVED** [review]

File: proto/executor/v1/executor.proto [hunk 1]
```diff
@@ -4,7 +4,7 @@ import "google/protobuf/empty.proto";
 
 package executor.v1;
 
-option go_package = "github.com/0xPolygonHermez/zkevm-node/state/runtime/executor";
+option go_package = "github.com/0xPolygonHermez/zkevm-node/state/runtime/executor/pb";
```
Comment (ToniRamirezM) [inline proto/executor/v1/executor.proto]: Do not make this change

**Review by ToniRamirezM — CHANGES_REQUESTED** [review]

File: proto/executor/v1/executor.proto [hunk 2]
```diff
@@ -319,24 +319,138 @@ enum ExecutorError {
...
+    EXECUTOR_ERROR_SM_MAIN_MULTIPLE_FREEIN = 33;
+    // EXECUTOR_ERROR_SM_MAIN_ASSERT indicates that the ROM assert instruction failed
+    EXECUTOR_ERROR_SM_MAIN_ASSERT = 34;
+    // EXECUTOR_ERROR_SM_MAIN_MEMORY indicates that the memory instruction check failed
+    EXECUTOR_ERROR_SM_MAIN_MEMORY = 35;
+    // EXECUTOR_ERROR_SM_MAIN_STORAGE_READ_MISMATCH indicates that the storage read instruction check failed
+    EXECUTOR_ERROR_SM_MAIN_STORAGE_READ_MISMATCH = 36;
+    // EXECUTOR_ERROR_SM_MAIN_STORAGE_WRITE_MISMATCH indicates that the storage read instruction check failed
+    EXECUTOR_ERROR_SM_MAIN_STORAGE_WRITE_MISMATCH = 37;
+    // EXECUTOR_ERROR_SM_MAIN_HASHK_VALUE_MISMATCH indicates that the Keccak hash instruction check failed
```
Comment (ToniRamirezM) [inline proto/executor/v1/executor.proto]: Errors 38, 39 y 40 have the same description

**Review by ToniRamirezM — CHANGES_REQUESTED** [review]

File: proto/executor/v1/executor.proto (hunk 2 above)
Comment (ToniRamirezM) [reply proto/executor/v1/executor.proto]: The same for 46, 47 and 48

File: proto/executor/v1/executor.proto (hunk 1 above)
Comment (fractasy) [reply proto/executor/v1/executor.proto]: Done

File: proto/executor/v1/executor.proto (hunk 2 above)
Comment (fractasy) [reply proto/executor/v1/executor.proto]: Fixed

**Comment by fractasy** [comment]
Using PR 88, which is a superset of this one.

**Pull request closed without merging**